tests/acceptance/features/webUIResharingToRoot/reshareUsers.feature

```diff
@@ -181,73 +187,74 @@ Feature: Resharing shared files with different permissions
-    Then as "Carol" folder "simple-folder (2)" should exist
-    And as "Carol" file "lorem (2).txt" should exist
+    And the user shares folder "simple-folder" with user "Carol King" as "Editor" using the webUI
+    And the user shares file "lorem.txt" with user "Carol King" as "Editor" using the webUI
+    Then as "Carol" folder "simple-folder" should exist
+    And as "Carol" file "lorem.txt" should exist
 
 
   Scenario: Resource owner sees resharer in collaborators list
-    Given user "Carol" has been created with default attributes
-    And user "Alice" has shared folder "simple-folder" with user "Brian"
-    And user "Brian" has shared folder "simple-folder (2)" with user "Carol"
-    When user "Alice" has logged in using the webUI
+    Given user "Brian" has shared folder "simple-folder" with user "Alice"
+    And user "Alice" has shared folder "simple-folder" with user "Carol"
+    When user "Brian" has logged in using the webUI
     And the user opens the share dialog for folder "simple-folder" using the webUI
-    Then user "Brian Murphy" should be listed as "Editor" in the collaborators list on the webUI
-    And user "Carol King" should be listed as "Editor" reshared through "Brian Murphy" in the collaborators list on the webUI
+    Then user "Alice Hansen" should be listed as "Editor" in the collaborators list on the webUI
+    And user "Carol King" should be listed as "Editor" reshared through "Alice Hansen" in the collaborators list on the webUI
 
 
   Scenario: Share recipient sees resharer in collaborators list
-    Given user "Carol" has been created with default attributes
-    And user "David" has been created with default attributes
+    Given user "David" has been created with default attributes and without skeleton files
     And group "Davidgrp" has been created
     And user "David" has been added to group "Davidgrp"
-    And user "Alice" has shared folder "simple-folder" with user "Brian"
-    And user "Alice" has shared folder "simple-folder" with user "Carol"
-    And user "Brian" has shared folder "simple-folder (2)" with user "David"
-    And user "Carol" has shared folder "simple-folder (2)" with group "Davidgrp"
+    And user "Brian" has shared folder "simple-folder" with user "Alice"
+    And user "Brian" has shared folder "simple-folder" with user "Carol"
+    And user "Alice" has shared folder "simple-folder" with user "David"
+    And user "Carol" has shared folder "simple-folder" with group "Davidgrp"
     When user "David" has logged in using the webUI
-    And the user opens the share dialog for folder "simple-folder (2)" using the webUI
-    Then user "Alice Hansen" should be listed as "Owner" reshared through "Brian Murphy, Carol King" in the collaborators list on the webUI
+    And the user opens the share dialog for folder "simple-folder" using the webUI
+    Then user "Brian Murphy" should be listed as "Owner" reshared through "Alice Hansen, Carol King" in the collaborators list on the webUI
```

Comment:
Maybe you don't need to change the owner from `Alice` to `Brian` 🤔 Other than that, everything looks good to me

Reply:
In `Background` I have created a folder for user `Brian` as  Brian is used to share resource in most of the tests there, so in order to use that I have changed some tests that use `Alice` to share resource.
That way, we can omit creating files folders for user Alice also